### PR TITLE
Feature/87 kinesis nested data support

### PR DESCRIPTION
**Pull request by AnastasiiaSergienko**

### CODE REVIEW

File: docs/kinesis/kinesis_connector.md [hunk 1]
```diff
@@ -58,6 +58,21 @@ CREATE OR REPLACE TABLE <schema_name>.<table_name>
...
+|----------------------|---------|---------------------------------|
+| Array                | *       | VARCHAR, CHAR                   |
+| Boolean              | Yes     | BOOLEAN                         |
+| Number               | Yes     | DECIMAL, DOUBLE PRECISION       |
+| Object (JSON Object) | *       | VARCHAR, CHAR                   |
+| String               | Yes     | VARCHAR, CHAR                   |
+
+Null values are supported.
+
+* Currently, the Connector has a flat data mapping and does not support mapping of the nested JSON Objects and Arrays.
```
Comment (morazow) [inline docs/kinesis/kinesis_connector.md:73]:
```suggestion
* Currently, the connector has a flat data mapping and does not support mapping of the nested JSON Objects and Arrays.
```

File: src/it/scala/com/exasol/cloudetl/kinesis/KinesisImportQueryGeneratorIT.scala [hunk 2]
```diff
@@ -63,8 +58,57 @@ class KinesisImportQueryGeneratorIT
...
+    currentTemperature: Int,
+    status: String,
+    partitionKey: String,
+    streamName: String
+  ): Unit = {
+    val recordData =
+      s"""{\"sensorId\": $sensorId,
+         | \"currentTemperature\": $currentTemperature,
+         | \"status\": \"$status\"
+         | }""".stripMargin.replace("\n", "")
```
Comment (morazow) [inline src/it/scala/com/exasol/cloudetl/kinesis/KinesisImportQueryGeneratorIT.scala:89]: You should be able to remove the escapes inside string interpolation. Could you please check it and remove `\"`?

File: src/it/scala/com/exasol/cloudetl/kinesis/KinesisImportQueryGeneratorIT.scala [hunk 3]
```diff
@@ -87,14 +131,85 @@ class KinesisImportQueryGeneratorIT
...
+  ): Unit = {
+    val partitionKey = "shardId-000000000000"
+    val recordData =
+      s"""{\"array_val\": [\"first\", \"second\"],
+         | \"bool_val\": true,
+         | \"int_val\": 10,
+         | \"double_val\": 10.55,
+         | \"object_val\": {\"firstNestedValue\": 10,\"secondNestedValue\": \"second\"},
+         | \"string_val\": null
+         | }""".stripMargin.replace("\n", "")
```
Comment (morazow) [inline src/it/scala/com/exasol/cloudetl/kinesis/KinesisImportQueryGeneratorIT.scala:166]: As above, remove escapes.

Comment (morazow) [reply src/it/scala/com/exasol/cloudetl/kinesis/KinesisImportQueryGeneratorIT.scala:166]: And all other places where string interpolation is used.

File: src/it/scala/com/exasol/cloudetl/kinesis/KinesisImportQueryGeneratorIT.scala (hunk 2 above)
Comment (AnastasiiaSergienko) [reply src/it/scala/com/exasol/cloudetl/kinesis/KinesisImportQueryGeneratorIT.scala:89]: it worked!

File: src/it/scala/com/exasol/cloudetl/kinesis/KinesisImportQueryGeneratorIT.scala (hunk 3 above)
Comment (AnastasiiaSergienko) [reply src/it/scala/com/exasol/cloudetl/kinesis/KinesisImportQueryGeneratorIT.scala:166]: done

File: src/it/scala/com/exasol/cloudetl/kinesis/KinesisImportQueryGeneratorIT.scala (hunk 2 above)
Comment (morazow) [reply src/it/scala/com/exasol/cloudetl/kinesis/KinesisImportQueryGeneratorIT.scala:89]: Great :+1:

**Pull request merged**